crates/rolldown/tests/rolldown/function/experimental/strict_execution_order/issue_4782/_config.json

```diff
@@ -8,5 +8,6 @@
     {
       "onDemandWrapping": true
     }
-  ]
+  ],
+  "expectExecuted": false
```

Comment:
temp disabled because https://github.com/rolldown/rolldown/blob/78047c4f61592046a4da000d41b2e0a30c36dbd1/crates/rolldown/src/stages/generate_stage/code_splitting.rs?plain=1#L252-L256, we will not ensure ensure_lazy_module_initialization_order when `strictExecutionOrder` is enabled.